src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -456,6 +527,95 @@ void CC32xxWiFi::wlan_mac(uint8_t mac[6])
     sl_NetCfgGet(SL_NETCFG_MAC_ADDRESS_GET, nullptr, &len, mac);
 }
 
+/*
+ * CC32xxWiFi::test_mode_start()
+ */
+void CC32xxWiFi::test_mode_start()
```

Comment:
how do we exit test mode?

Reply:
normal CC32xxWiFi::stop(), added comment.